app/src/main/cpp/skyline/services/serviceman.h

```diff
@@ -10,49 +10,46 @@
 namespace skyline::service {
     /**
      * @brief The ServiceManager class manages passing IPC requests to the right Service and running event loops of Services
+     * @todo This implementation varies significantly from HOS, this should be rectified much later on
```

Comment:
Remove this, we don't want such TODOs floating around in the codebase forever

Reply:
Nah we will need to do it, so it should be kept as it explains our inaccuracies